bentoml/xgboost.py

```diff
@@ -32,22 +31,32 @@
     )
 
 
-def _save_model(
+def save(
```

Comment:
@parano we will enable context options to pass per framework implementation, but all of them can have the context of function call via `sys._getframe().f_back.f_code.co_filename`